care.config.ts

```diff
@@ -103,6 +105,12 @@ const careConfig = {
   abdm: {
     enabled: (env.REACT_ENABLE_ABDM ?? "true") === "true",
   },
+
+  scribe: {
+    enabled: env.REACT_ENABLE_SCRIBE === "true",
+  },
```

Comment:
We moved to using backend flags, why add a switch from FE as well 😅

Reply:
Fixed it, caused due to a bad conflict resolution